packages/test-subj-selector/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@spalger/test-subj-selector",
```

Comment:
`@elastic/test-subj-selector`

We should also consider renaming to `@kbn/test-subj-selector` (we've been gradually doing this while moving packages in here)

Reply:
Btw, we've planned to adding a check that verifies that the name in

```
"@elastic/test-subj-selector": "link:packages/test-subj-selector"
```

matches the `name` in the `link:`'d `package.json`. We just haven't had time to add it yet.

Reply:
Should I rename to `@elastic/test-subj-selector` or `@ kbn/test-subj-selector`?

Reply:
Yeah, I'd prefer renaming.